dataduct/steps/transform.py

```diff
@@ -53,20 +57,18 @@ def __init__(self,
             raise ETLInputError(
                 'Only one of script, command and directory allowed')
 
-        base_output_node = None
-        if not no_output:
-            # Create output_node based on output_path
-            base_output_node = self.create_s3_data_node(
-                self.get_output_s3_path(get_modified_s3_path(output_path)))
+        # Create output_node based on output_path
+        base_output_node = self.create_s3_data_node(
+            self.get_output_s3_path(get_modified_s3_path(output_path)))
```

Comment:
why remove the if as it will create a useless S3Node, doesn't make an actual difference